react/features/app-settings/components/AbstractAppSettings.js

```diff
@@ -10,6 +10,16 @@ import { getProfile, updateProfile } from '../../base/profile';
 */
 type Props = {
 
+    /**
+    * The current aspect ratio of the screen.
+    */
```

Comment:
I'd like to please attract your attention to the formatting of the multiline comments. I presume it's an editor setting so could you please make sure you comply with our coding style?

We don't do:
````javascript
/**
*
*/
````

We do:
````javascript
/**
 *
 */
````